CHANGELOG.md

```diff
@@ -5,6 +5,189 @@ information on the list of deprecated flags and APIs please have a look at
 https://docs.docker.com/engine/deprecated/ where target removal dates can also
 be found.
 
+## 1.13.0 (2016-12-08)
+
+### Builder
++ Add capability to specify images used as a cache source on build. These images do not need to have local parent chain and can be pulled from other registries [#26839](https://github.com/docker/docker/pull/26839)
++ (experimental) Add option to squash image layers to the FROM image after successful builds [#22641](https://github.com/docker/docker/pull/22641)
+* Fix dockerfile parser with empty line after escape [#24725](https://github.com/docker/docker/pull/24725)
+- add step # on `docker build` [#24978](https://github.com/docker/docker/pull/24978)
++ Add support for compressing build context during image build [#25837](https://github.com/docker/docker/pull/25837)
++ add `--network` to `docker build` [#27702](https://github.com/docker/docker/pull/27702)
+- Fix inconsistent behavior between `--label` flag on `docker build` and `docker run` [#26027](https://github.com/docker/docker/issues/26027)
+- Fix image layer inconsistencies when using the overlay storage driver [#27209](https://github.com/docker/docker/pull/27209)
+* Unused build-args are now allowed. A warning is presented instead of an error and failed build [#27412](https://github.com/docker/docker/pull/27412)
+- Fix builder cache issue on Windows [#27805](https://github.com/docker/docker/pull/27805)
+
+### CLI
+
++ Add `--mount` flag to `docker run` [#26825](https://github.com/docker/docker/pull/26825)[#28150](https://github.com/docker/docker/pull/28150)
++ Add `--format` flag to `network ls` and `volume ls` [#23475](https://github.com/docker/docker/pull/23475)
+* Allow the top-level `docker inspect` command to inspect any kind of resource [#23614](https://github.com/docker/docker/pull/23614)
+- Allow unsetting the `--entrypoint` in `docker run` or `docker create` [#23718](https://github.com/docker/docker/pull/23718)
+- Sanitize docker labels when used as journald field names [#23725](https://github.com/docker/docker/pull/23725)
+* Improve performance when "killing", or removing multiple containers by performing these tasks in parallel [#26778](https://github.com/docker/docker/pull/26778)
```

Comment:
Why is this in `CLI` and "allow parallel stop, pause, unpause" in `Runtime`. Essentially they should be same bullet point with 2 PR links(improve stopping, removing and pausing).